drivers/telescope/lx200_10micron.cpp

```diff
@@ -784,19 +757,30 @@ bool LX200_10MICRON::CalculateTrajectory(int year, int month, int day, int hour,
     // mark the beginning and the end of the given transit. Flags is a string which can be
     // empty or contain the letter F – meaning that mount will flip during the transit.
     // Available from version 2.13.20.
-    LOGF_INFO("Calculate trajectory is called with date: %d-%d-%d %d:%d pass %d",
-                year, month, day, hour, minute, nextpass);
-    date_pass.years = year;
-    date_pass.months = month;
-    date_pass.days = day;
-    date_pass.hours = hour;
-    date_pass.minutes = minute;
-    date_pass.seconds = 0.0;
-    JD = ln_get_julian_day(&date_pass);
+    struct ln_date start_pass;
+    if (extractISOTime(start_pass_iso, &start_pass) == -1)
```

Comment:
does this thing require HAVE_LIBNOVA ?

indicom.h has it in #ifndef _WIN32
indicom.c has it in #if defined(HAVE_LIBNOVA)

Reply:
It does require HAVE_LIBNOVA, in order to use `ln_date` and `ln_get_julian_day`. What should I do in this respect?

Reply:
I think the `#ifndef _WIN32` in indicom.h should be replaced with `#if defined(HAVE_LIBNOVA)` and the extractISOTime calls should have `#if defined(HAVE_LIBNOVA)` as well. @knro do you agree ?

Reply:
I can do that, it's just two lines. I am just unsure what would be the behavior in that case: if it does not have LIBNOVA it should simply return an error `LOG_ERROR("Libnova not available, can't process date")` or something like that? Or do you have some other suggestion?

Reply:
No need. As I said, all drivers have access to libnova.